src/components/ModalConfirmation.js

```diff
@@ -0,0 +1,35 @@
+import Modal from "./Modal.js";
+
+export default class ModalConfirmation extends Modal {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.

Reply:
Awesome, thanks!